backend/src/BuiltinExecution/Libs/Base64.fs

```diff
@@ -81,22 +87,54 @@ let fns : List<BuiltInFn> =
 
     { name = fn "urlEncode" 0
       typeParams = []
-      parameters = [ Param.make "bytes" TBytes "" ]
+      parameters = [ Param.make "bytes" (TList(TUInt8)) "" ]
       returnType = TString
       description =
         "Base64URL encodes <param bytes> with {{=}} padding. Uses URL-safe encoding
          with {{-}} and {{_}} instead of {{+}} and {{/}}, as defined in RFC 4648
          section [5](https://www.rfc-editor.org/rfc/rfc4648.html#section-5)."
       fn =
         (function
-        | _, _, [ DBytes bytes ] ->
+        | _, _, [ DList(_vt, bytes) ] ->
+          let bytes = Dval.DlistToByteArray bytes
           // Differs from Base64.encodeToUrlSafe as this version has padding
           System.Convert.ToBase64String(bytes).Replace('+', '-').Replace('/', '_')
           |> DString
           |> Ply
         | _ -> incorrectArgs ())
       sqlSpec = NotYetImplemented
       previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "hexEncode" 0
```

Comment:
Not sure if this is the right place for this function. It was originally in 'Bytes.fs' and is used in both crypto and big tests

Reply:
I'd maybe put these in bytes? It feels like this could go into a package though?

Reply:
I moved them to Bytes for now, I'll see if I can move it to a package